### DISCUSSION TITLE
How prevent VC to displau the name of the note we are in ?

### DISCUSSION THREAD

**Discussion by Psophos44** [tadashi-aikawa/obsidian-various-complements-plugin#251] [Q&A]
Dear Tadashi,

I love your Obsidian plugin Various complements which help to create new links between notes.
But here is my question : when I'm writing into an obsidian note, the plugin helps me by proposing some notes to link with (that's perfect) but also proposes to link to the current note when I use one word of the title of the current note.
Of course, there is no sense to link a note to itself...

Could you explain me how to do to prevent VC to show me the link to the current note ?

Thank you very much and bravo for this awsome plugin.

Olivier (France)

**Comment by tadashi-aikawa** [comment]
Hi, @Psophos44 :)

> Could you explain me how to do to prevent VC to show me the link to the current note ?

No, there is not. The feature request is https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues/134.

> Of course, there is no sense to link a note to itself...

I think it is not necessarily so. For example, in the case of a term note, by referring to itself, when the note title is changed, the name in the note is also automatically changed.
